modules/back-end/deploy/Dockerfile

```diff
@@ -27,14 +27,20 @@ RUN dotnet publish -c Release --no-build -o /app
 
 # final stage/image
 FROM mcr.microsoft.com/dotnet/aspnet:6.0
+
+RUN apt-get update && apt-get install -y curl unzip && rm -rf /var/lib/apt/lists/*
+ARG OTEL_VERSION=1.0.2
+ADD https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/download/v${OTEL_VERSION}/otel-dotnet-auto-install.sh otel-dotnet-auto-install.sh
```

Comment:
How about use the latest version? Like [here,](https://opentelemetry.io/docs/languages/net/automatic/#linux-and-macos) it uses this link: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/latest/download/otel-dotnet-auto-install.sh